woodchipper/http/fastapi.py

```diff
@@ -61,7 +49,7 @@ async def send_with_extra_headers(message):
         with LoggingContext(
             "fastapi:request",
             **{
-                "id": self._request_id_factory(),
+                "id": self._request_id_factory() if self._request_id_factory is not None else str(uuid.uuid4()),
```

Comment:
Why move this away from the init?

Reply:
Just for consistency in how `chipperize` works vs. the middleware-attach method. In `chipperize` we pass the same arguments passed to the constructor (or their defaults). That default is `None`, not a lambda.